cremalab/crema-app-native
# Upgrade Expo SDK to v38

## Description

Upgraded Expo SDK to v38

I ensured that:
- the app runs on Android
- the app runs on iOS
- unit tests run and pass
- storybook runs on Android
- storybook runs on iOS
- loki runs on Android
- loki runs on iOS
- test:ci passes

### 👩‍🔬 Test Instructions

1. Pull down PR
1. Run `npm ci`
1. Ensure app starts and runs

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


---

>### Roles & Responsibilities
>
>#### 👨‍💻 Assignee
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Reviewer
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
